### ISSUE TITLE
Error: Access is denied. (os error 5) on windows.

### ISSUE THREAD

**Issue by laundmo** [cargo-generate/cargo-generate#634]
When running cargo generate with a Git path or URL on Windows 10 (19042) i get the following error:

`cargo generate ashleygwilliams/wasm-pack-template` (using the example template from the readme)
```
⚠️   Favorite `ashleygwilliams/wasm-pack-template` not found in config, using it as a git repository: https://github.com/ashleygwilliams/wasm-pack-template.git
💡   Using private key: `\\?\%userprofile%\.ssh\id_rsa` for git-ssh checkout
Error: Access is denied. (os error 5)
```

The same happens when using `cargo generate https://github.com/ashleygwilliams/wasm-pack-template.git`, or when running the command with elevated permissions.

When cloning the template first and then using the `--path` flag, i'm able to generate the project.

**Comment by tjf801** [comment]
I am currently getting this same error.

**Comment by sassman** [comment]
this is very likely a duplicate of #630

**Comment by sassman** [comment]
could you please also check if you have something like this in your `~/.gitconfig`?

```
[url "git@github.com:"]
    insteadOf = https://github.com/
```

**Comment by sassman** [comment]
this should be fixed with `v0.15.0`.
